src/modules/tests/test-data/cat-adi-part2/vehicle-checks/vehicle-checks.cat-adi-part2.action.ts

```diff
@@ -12,12 +12,22 @@ export const VEHICLE_CHECKS_REMOVE_SERIOUS_FAULT = '[Vehicle Checks] [CatADI2] V
 export const VEHICLE_CHECKS_ADD_DANGEROUS_FAULT = '[Vehicle Checks] [CatADI2] Vehicle Checks Add Dangerous Fault';
 export const VEHICLE_CHECKS_REMOVE_DANGEROUS_FAULT = '[Vehicle Checks] [CatADI2] Vehicle Checks Remove Dangerous Fault';
 export const ADD_SHOW_ME_TELL_ME_COMMENT = '[Vehicle Checks] [CatADI2] Add Show Me / Tell Me comment';
+export const VEHICLE_CHECKS_COMPLETED_SELECTED = '[Vehicle Checks] [CatADI2] Vehicle Checks Completed Selected';
+export const VEHICLE_CHECKS_COMPLETED_DESELECTED = '[Vehicle Checks] [CatADI2] Vehicle Checks Completed Deselected';
 
 export class ShowMeQuestionSelected implements Action {
   readonly type = SHOW_ME_QUESTION_SELECTED;
   constructor(public showMeQuestion: QuestionResult, public index: number) {}
 }
 
+export class VehicleChecksCompletedSelected implements Action {
```

Comment:
can we have a single action for consistency with other toggle buttons, eg. TOGGLE_VEHICLE_CHECKS which just inverts the value? See other test report toggles like src/modules/tests/test-data/common/eta/eta.reducer.ts